### PR TITLE
feat(Styles): Port SCSS utilities to TS

### PR DESCRIPTION
**Pull request by codecaaron**
### PR Checklist

- [ ] Related to Abstract designs:
- [ ] Related to JIRA ticket: ABC-123
- [ ] I have run this code to verify it works
- [ ] This PR includes unit tests for the code change

### CODE REVIEW

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-labs@1.8.3-alpha.18d4d1.0<br/>@codecademy/gamut-styles@4.2.1-alpha.18d4d1.0<br/>@codecademy/gamut@12.4.1-alpha.18d4d1.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5f28f1bb903f165fb2af3dd2--gamut-preview.netlify.app>https://5f28f1bb903f165fb2af3dd2--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/5f28f1bb903f165fb2af3dd2>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-labs@1.8.3-alpha.6667d3.0<br/>@codecademy/gamut-styles@4.2.1-alpha.6667d3.0<br/>@codecademy/gamut@12.4.1-alpha.6667d3.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5f298b16aeb2723067fc1ec2--gamut-preview.netlify.app>https://5f298b16aeb2723067fc1ec2--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/5f298b16aeb2723067fc1ec2>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-labs@1.8.3-alpha.02e28c.0<br/>@codecademy/gamut-styles@4.2.1-alpha.02e28c.0<br/>@codecademy/gamut@12.4.1-alpha.02e28c.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5f299fd1e94e4a598bc1ef44--gamut-preview.netlify.app>https://5f299fd1e94e4a598bc1ef44--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/5f299fd1e94e4a598bc1ef44>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-labs@1.8.3-alpha.404da5.0<br/>@codecademy/gamut-styles@4.2.1-alpha.404da5.0<br/>@codecademy/gamut@12.4.1-alpha.404da5.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5f29bc39a56d167840bf1ddd--gamut-preview.netlify.app>https://5f29bc39a56d167840bf1ddd--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/5f29bc39a56d167840bf1ddd>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-labs@1.8.3-alpha.44316a.0<br/>@codecademy/gamut-styles@4.2.1-alpha.44316a.0<br/>@codecademy/gamut@12.4.1-alpha.44316a.0<br/></pre>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-labs@1.8.3-alpha.307303.0<br/>@codecademy/gamut-styles@4.2.1-alpha.307303.0<br/>@codecademy/gamut@12.4.1-alpha.307303.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5f29c86fd6a50e0283ca4d81--gamut-preview.netlify.app>https://5f29c86fd6a50e0283ca4d81--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/5f29c86fd6a50e0283ca4d81>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-labs@1.8.3-alpha.6a96e9.0<br/>@codecademy/gamut-styles@4.2.1-alpha.6a96e9.0<br/>@codecademy/gamut@12.4.1-alpha.6a96e9.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5f29d3953591d36ed40ceca5--gamut-preview.netlify.app>https://5f29d3953591d36ed40ceca5--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/5f29d3953591d36ed40ceca5>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-labs@1.8.3-alpha.7a8239.0<br/>@codecademy/gamut-styles@4.2.1-alpha.7a8239.0<br/>@codecademy/gamut@12.4.1-alpha.7a8239.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5f2a351a2ef8f226df58645e--gamut-preview.netlify.app>https://5f2a351a2ef8f226df58645e--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/5f2a351a2ef8f226df58645e>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-labs@1.8.3-alpha.df23af.0<br/>@codecademy/gamut-styles@4.2.1-alpha.df23af.0<br/>@codecademy/gamut@12.4.1-alpha.df23af.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5f2aaa2a482388a669443c06--gamut-preview.netlify.app>https://5f2aaa2a482388a669443c06--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/5f2aaa2a482388a669443c06>Deploy Logs</a></p>

File: packages/gamut-styles/src/variables/colors.ts [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+  black: '#000000',
+  white: '#ffffff',
+  red: '#fd4d3f',
+  orange: '#ff9b2a',
+  yellow: '#ffc740',
+  purple: '#6400e4',
+  pink: '#ff8ef5',
+  mint: '#6df0c2',
+  beige: '#efd9ca',
+  darkBlue: '#141c3a',
```
Comment (codecaaron) [inline packages/gamut-styles/src/variables/colors.ts:106]: Not sure what we want to do about the redundancy here.

File: packages/gamut-styles/src/variables/responsive.ts [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+export const breakpoints: Record<MediaSize, string> = {
+  xs: '480px',
+  sm: '768px',
+  md: '1024px',
+  lg: '1200px',
+  xl: '1440px',
+};
+
+const createMediaQuery = (size: MediaSize, direction: 'min' | 'max') => `
```
Comment (jakemhiller) [inline packages/gamut-styles/src/variables/responsive.ts:11]: I like that this isn’t exported, but i also assume we’ll have to at some point. Just thinking out loud, i think it’s good to keep it like this for now

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-labs@1.8.3-alpha.2a3647.0<br/>@codecademy/gamut-styles@4.2.1-alpha.2a3647.0<br/>@codecademy/gamut@12.4.1-alpha.2a3647.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5f2c4b9aa05a9ac3f23a9e0b--gamut-preview.netlify.app>https://5f2c4b9aa05a9ac3f23a9e0b--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/5f2c4b9aa05a9ac3f23a9e0b>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-labs@1.8.3-alpha.8cde81.0<br/>@codecademy/gamut-styles@4.2.1-alpha.8cde81.0<br/>@codecademy/gamut@12.4.1-alpha.8cde81.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5f2c5f15b96dd41110f73314--gamut-preview.netlify.app>https://5f2c5f15b96dd41110f73314--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/5f2c5f15b96dd41110f73314>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-labs@1.8.3-alpha.0d2daa.0<br/>@codecademy/gamut-styles@4.2.1-alpha.0d2daa.0<br/>@codecademy/gamut@12.4.1-alpha.0d2daa.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5f2c840ea1c2de742b484089--gamut-preview.netlify.app>https://5f2c840ea1c2de742b484089--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/5f2c840ea1c2de742b484089>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-labs@1.8.3-alpha.9e4048.0<br/>@codecademy/gamut-styles@4.2.1-alpha.9e4048.0<br/>@codecademy/gamut@12.4.1-alpha.9e4048.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5f2d7f060ee932fdcf929bde--gamut-preview.netlify.app>https://5f2d7f060ee932fdcf929bde--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/5f2d7f060ee932fdcf929bde>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-labs@1.8.3-alpha.6664a4.0<br/>@codecademy/gamut-styles@4.2.1-alpha.6664a4.0<br/>@codecademy/gamut@12.4.1-alpha.6664a4.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5f2d801d0023802687261985--gamut-preview.netlify.app>https://5f2d801d0023802687261985--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/5f2d801d0023802687261985>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-labs@1.8.3-alpha.4f741e.0<br/>@codecademy/gamut-styles@4.2.1-alpha.4f741e.0<br/>@codecademy/gamut@12.4.1-alpha.4f741e.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5f2d972d3b27420167f437b1--gamut-preview.netlify.app>https://5f2d972d3b27420167f437b1--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/5f2d972d3b27420167f437b1>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-labs@1.8.3-alpha.bfcc0a.0<br/>@codecademy/gamut-styles@4.2.1-alpha.bfcc0a.0<br/>@codecademy/gamut@12.4.1-alpha.bfcc0a.0<br/></pre>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-labs@1.8.3-alpha.a7dabe.0<br/>@codecademy/gamut-styles@4.2.1-alpha.a7dabe.0<br/>@codecademy/gamut@12.4.1-alpha.a7dabe.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5f2da60fd88d1fe0ee2c028d--gamut-preview.netlify.app>https://5f2da60fd88d1fe0ee2c028d--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/5f2da60fd88d1fe0ee2c028d>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5f2da645d4b6d5d08facbf8e--gamut-preview.netlify.app>https://5f2da645d4b6d5d08facbf8e--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/5f2da645d4b6d5d08facbf8e>Deploy Logs</a></p>

File: jest.config.js [hunk 3]
```diff
@@ -23,6 +23,8 @@ module.exports = {
     '\\.(jpg|jpeg|png|gif|eot|otf|webp|svg|ttf|woff|woff2|mp4|webm|wav|mp3|m4a|aac|oga|md)$':
       '<rootDir>/script/jest/fileMock',
     '\\.(css|scss)$': '<rootDir>/script/jest/styleMock',
+    '^~styleguide/Blocks(.*)$':
+      '<rootDir>/packages/styleguide/.storybook/Blocks$1',
```
Comment (codecaaron) [inline jest.config.js:27]: I had to move Storybook native components to `./storybook` to ensure that they would be transpiled prior to stories.

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-labs@1.8.3-alpha.152fe5.0<br/>@codecademy/gamut-styles@4.2.1-alpha.152fe5.0<br/>@codecademy/gamut@12.4.1-alpha.152fe5.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5f2db6793b27427a70f439e7--gamut-preview.netlify.app>https://5f2db6793b27427a70f439e7--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/5f2db6793b27427a70f439e7>Deploy Logs</a></p>

File: packages/gamut-styles/src/variables/colors.ts (hunk 1 above)
Comment (kipprice) [reply packages/gamut-styles/src/variables/colors.ts:106]: pull from the above constants instead?

File: packages/gamut-styles/src/variables/typography.ts [hunk 4]
```diff
@@ -0,0 +1,62 @@
+import { colors as colorSwatch } from './colors';
+import { pxRem } from '../utilities/pxRem';
+import { base } from './base';
+
+export const fontBase = `"Nunito Sans", -apple-system, BlinkMacSystemFont, "Segoe UI",
```
Comment (kipprice) [inline packages/gamut-styles/src/variables/typography.ts:5]: slight preference for specifying `fontFamily` instead of just `font`

File: packages/gamut-styles/src/variables/typography.ts [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+-apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Ubuntu", "Cantarell",
+"Fira Sans", "Droid Sans", "Helvetica Neue", sans-serif`;
+
+export const fontMonospace = `Monaco, Menlo, "Ubuntu Mono", "Droid Sans Mono", Consolas,
+monospace`;
+
+export const fontSystem = `-apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Ubuntu",
+"Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue", sans-serif`;
+
+export const fontStack = {
```
Comment (kipprice) [inline packages/gamut-styles/src/variables/typography.ts:19]: similar to above, slight preference for `fontFamilies` over `fontStack`

File: packages/gamut-styles/src/variables/typography.ts [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+    xxl: pxRem(base * 3),
+    xl: pxRem(base * 2.2),
+    lg: pxRem(base * 1.6),
+    md: pxRem(base * 1.4),
+    sm: pxRem(base * 1.25),
+    xs: pxRem(base),
+  },
+};
+
+export const lineHeight = {
```
Comment (kipprice) [inline packages/gamut-styles/src/variables/typography.ts:54]: can this be named with the expected units? (e.g. `lineHeightInRems`)

File: packages/gamut-styles/src/variables/typography.ts [hunk 7]
```diff
@@ -0,0 +1,62 @@
...
+  link: colorSwatch.blue[500],
+  linkHover: colorSwatch.blue[700],
+};
+
+export const fontDecoration = {
+  link: 'none',
+  linkHover: 'underline',
+};
+
+export const fontSize = {
```
Comment (kipprice) [inline packages/gamut-styles/src/variables/typography.ts:38]: does `px-rem` return a css val string (e.g. `16px`) or the raw number? If it's the latter, I prefer the name `fontSizeInPx` for the exported const

File: packages/gamut-styles/utils/variables/_grid.scss [hunk 8]
```diff
@@ -1,13 +1,14 @@
...
 $grid-gutter-width: px-rem(16);
 $grid-outer-margin: px-rem(16);
 $grid-xs-min: px-rem(480);
 $grid-sm-min: px-rem(768);
 $grid-md-min: px-rem(1024);
 $grid-lg-min: px-rem(1200);
-$grid-xl-min: px-rem($max-content-width);
+$grid-xl-min: px-rem(1440);
+
+$max-content-width: $grid-lg-min;
```
Comment (kipprice) [inline packages/gamut-styles/utils/variables/_grid.scss:13]: these seem like maybe they're reversed? `max-content-width` is smaller than `medium-content-width`

Comment (kipprice) [reply packages/gamut-styles/utils/variables/_grid.scss:13]: if the values are accurate, can we update the var names to make clear what these should be used for?

File: packages/gamut/src/Badge/index.tsx [hunk 9]
```diff
@@ -1,12 +1,15 @@
...
+  colors,
+  brandColors,
+  fontSize,
+} from '@codecademy/gamut-styles';
 
 export const Badge = styled.span`
   background: ${colors.blue[500]};
   border-radius: 3px;
-  color: ${colors.white};
+  color: ${brandColors.white};
```
Comment (kipprice) [inline packages/gamut/src/Badge/index.tsx:10]: no particular feelings here, but why the change?

File: packages/gamut-styles/src/utilities/boxShadow.ts [hunk 10]
```diff
@@ -0,0 +1,31 @@
+import { css } from '@emotion/core';
+
+export type ShadowDepths = 1 | 2 | 3 | 4 | 5;
+
+export const boxShadow = (depth = 1 as ShadowDepths) => {
```
Comment (kipprice) [inline packages/gamut-styles/src/utilities/boxShadow.ts:5]: this could be made a little easier to read with a dictionary of shadowDepth -> unique numeric pair; something like:
```javascript
const shadowsForDepth = {
  1: [1, 3, 1, 3],
  2: [3, 6, 3, 6],
  3: [10, 20, 6, 6]
  ...
}

export const boxShadow = (depth = 1 as ShadowDepths) => {
  const [s1, s2, s3, s4] = shadowsForDepth[depth]
  return css`box-shadow: 0 ${s1}px ${s2}px rgba(0,0,0,0.16), 0 ${s3}px ${s4}px rgba(0,0,0,0.23);`;
}
```

File: packages/styleguide/stories/Core/Foundations/Utilities/Utilities.stories.mdx [hunk 11]
```diff
@@ -0,0 +1,176 @@
...
+
+.unselectable {
+  @include font-smoothing();
+}
+```
+
+### Screen Readers
+
+Ensuring that your content is accessible can require you to make screen reader only text elements.
+We provide a few helpful mixins for use cases where you want the screen reader to be able to focus or
```
Comment (kipprice) [inline packages/styleguide/stories/Core/Foundations/Utilities/Utilities.stories.mdx:144]: could you add an example of when you may use one of these? (e.g. hidden h1 for context)

File: packages/styleguide/stories/Core/Foundations/Utilities/Utilities.stories.mdx [hunk 12]
```diff
@@ -0,0 +1,176 @@
...
+}
+
+.rough {
+  @include font-smoothing(false);
+}
+```
+
+### No Selection
+
+In cases where we want to prevent the user from selecting an image or
```
Comment (kipprice) [inline packages/styleguide/stories/Core/Foundations/Utilities/Utilities.stories.mdx:119]: probably want to finish this >.<

File: packages/styleguide/stories/Core/Foundations/Utilities/Utilities.stories.mdx [hunk 13]
```diff
@@ -0,0 +1,176 @@
...
+@import '@codecademy/gamut-styles/utils';
+
+.box {
+  @include box-shadow(3);
+}
+```
+
+### `fontSmoothing`
+
+In some cases we may want to toggle font smoothing.
```
Comment (kipprice) [inline packages/styleguide/stories/Core/Foundations/Utilities/Utilities.stories.mdx:87]: are there examples in our code where we do enable font smoothing? I'm not confident when I would use this option at Codecademy.

File: packages/styleguide/stories/Core/Foundations/Utilities/Utilities.stories.mdx [hunk 14]
```diff
@@ -0,0 +1,176 @@
...
+}
+```
+
+## Style Mixins
+
+We also support various style partials to help you consistently achieve affects, like box shadows and screen reader content.
+
+### Box Shadow
+
+We have 5 depths of box shadows that we use.
```
Comment (kipprice) [inline packages/styleguide/stories/Core/Foundations/Utilities/Utilities.stories.mdx:47]: this seems like an opportunity to share any guidelines design uses for when to use types of shadows.

Comment (kipprice) [reply packages/styleguide/stories/Core/Foundations/Utilities/Utilities.stories.mdx:47]: also maybe less relevant in the new design system?

File: packages/styleguide/stories/Core/Foundations/Utilities/Utilities.stories.mdx [hunk 15]
```diff
@@ -0,0 +1,176 @@
...
+
+<StoryStatus status="volatile" />
+
+## Converting Units
+
+### PX to REM
+
+In order to keep consistent sizing we like to use `rem` units for most things (within reason).
+Many times when a specific REM values is not expressive like `1.875rem` or `30px` (relative to our base of 16)
+it is better to express this as a pixel value that is closer to a round number.
```
Comment (kipprice) [inline packages/styleguide/stories/Core/Foundations/Utilities/Utilities.stories.mdx:18]:
```suggestion
Many times when a specific REM values is not expressive like `1.875rem` 
it is better to express this as a pixel value (in this case, `30px`) that is closer to a round number.
```

File: packages/gamut-styles/src/utilities/boxShadow.ts (hunk 10 above)
Comment (codecaaron) [reply packages/gamut-styles/src/utilities/boxShadow.ts:5]: Nice optimization, I'll switch.

File: packages/gamut-styles/src/variables/typography.ts (hunk 7 above)
Comment (codecaaron) [reply packages/gamut-styles/src/variables/typography.ts:38]: It returns a rem value for a non nice PX value, I actually will probably switch these to pure rem values, but create a modular scale calculator for this going forward.

File: packages/gamut-styles/utils/variables/_grid.scss (hunk 8 above)
Comment (codecaaron) [reply packages/gamut-styles/utils/variables/_grid.scss:13]: Good catch, sorry!  Yeah naming is a work in progress on these, they are mostly used for content container for larger than 1440 screens.

File: packages/gamut/src/Badge/index.tsx (hunk 9 above)
Comment (codecaaron) [reply packages/gamut/src/Badge/index.tsx:10]: Will likely revert this.  I was trying to figure out a better way of separating solid colors vs color gradients.  But brand doesn't feel right here.

File: packages/styleguide/stories/Core/Foundations/Layout/Layout.stories.mdx [hunk 16]
```diff
@@ -0,0 +1,149 @@
...
+<Meta title="Foundations/Layout" component="Layout" />
+
+# Layout
+
+<StoryStatus status="volatile" />
+
+## Screensizes and Media Queries
+
+We have 5 specific screens that we design for. Ideally we do not deviate in design or code
+for these specific sizes to avoid unintended side effects. Here are there relative values.
```
Comment (kipprice) [inline packages/styleguide/stories/Core/Foundations/Layout/Layout.stories.mdx:16]:
```suggestion
for these specific sizes to avoid unintended side effects. Here are their relative values.
```

File: packages/styleguide/stories/Core/Foundations/Layout/Layout.stories.mdx [hunk 17]
```diff
@@ -0,0 +1,149 @@
...
+`;
+```
+
+However we do provide specific mobile first media queries out of the box to make this even easier with `mediaQueries`:
+
+```ts
+import { mediaQueries } from '@codecademy/gamut-styles';
+
+const MyStyles = `
+   ${mediaQueries.sm} {
```
Comment (kipprice) [inline packages/styleguide/stories/Core/Foundations/Layout/Layout.stories.mdx:47]:
```suggestion
  ${mediaQueries.sm} {
```

File: packages/styleguide/stories/Core/Foundations/Layout/Layout.stories.mdx [hunk 18]
```diff
@@ -0,0 +1,149 @@
...
+`;
+```
+
+#### SCSS - \$max-content-width
+
+```scss
+@import '@codecademy/gamut-styles/utils';
+
+.container {
+  max-width: $max-content-width;
```
Comment (kipprice) [inline packages/styleguide/stories/Core/Foundations/Layout/Layout.stories.mdx:120]: part of this change added mediumContentWidth, right?

File: packages/styleguide/stories/Core/Foundations/Typography/Typography.stories.mdx [hunk 19]
```diff
@@ -1,130 +1,186 @@
...
+
+<Meta title="Foundations/Typography" component="Typography" />
+
+# Typography
+
+<StoryStatus status="volatile" />
+
+This page is meant as a reference for Typography variables and scales that are available
+through both **Typescript** and **SCSS** variables in [`@codecademy/gamut-styles`](https://www.npmjs.com/package/@codecademy/gamut-styles). We heavily recommend
+you use the <LinkTo kind="Typography" story="Heading">`<Heading />`</LinkTo> and <LinkTo kind="Typography" story="Text">`<Text />`</LinkTo>
```
Comment (kipprice) [inline packages/styleguide/stories/Core/Foundations/Typography/Typography.stories.mdx:20]: can we make this more visible? (does mdx support ==highlighting==?)

Comment (kipprice) [reply packages/styleguide/stories/Core/Foundations/Typography/Typography.stories.mdx:20]: update: it does not support ==highlighting==. maybe a block quote instead?

Comment (codecaaron) [reply packages/styleguide/stories/Core/Foundations/Typography/Typography.stories.mdx:20]: I'm going to see if I can swap around some default MDX components to something more readable.

File: jest.config.js (hunk 3 above)
Comment (JoshuaKGoldberg) [reply jest.config.js:27]: @codecaaron Would you mind splitting these changes into a separate PR to keep this one smaller?

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-labs@1.8.3-alpha.62d275.0<br/>@codecademy/gamut-styles@4.2.1-alpha.62d275.0<br/>@codecademy/gamut@12.4.1-alpha.62d275.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5f34671187cbf1728d9b6c25--gamut-preview.netlify.app>https://5f34671187cbf1728d9b6c25--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/5f34671187cbf1728d9b6c25>Deploy Logs</a></p>

**Comment by codecaaron** [comment]
Closing in favor of https://github.com/Codecademy/client-modules/pull/920

Feedback is addressed in that new PR
